### DISCUSSION TITLE
Unable to use Matic bridge since Aug 6 2021

### DISCUSSION THREAD

**Discussion by farmernc505** [0xPolygon/matic.js#255] [General]
Unable to use Matic bridge since Aug 6 2021 - I can transfer from my Metamask to ETH Network fine but cannot
deposit into MATIC or out of MATIC. Need Help

**Comment by ujjwalguptaofficial** [comment]
are you trying to transfer by code ? or using UI from walllet.matic.network ? @farmernc505
